addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -66,19 +66,14 @@ $o-wblog-loader-size: 50px;
         background-color: rgba(black, 0.005);
     }
 
-    .o_wblog_read_with_sidebar {
-        max-width: map-get($container-max-widths, md) + $o-wblog-sidebar-width;
-    }
-
     // This option class is assigned to the post's content using the "Customize"
-    // menu. The aim is to be able to write simple articles on the fly,
-    // achieving a good design without being forced to use snippets.
+    // menu. The aim is to be able to write simple articles on the fly.
     .o_wblog_read_text {
```

Comment:
Is that class / option still useful? Can't remember why, can you explain?

Reply:
`. o_wblog_read_text` this class define some css rules for titles, captions, paragraphs ...